STL_Extension/include/CGAL/Modifiable_priority_queue.h

```diff
@@ -80,35 +44,50 @@ class Modifiable_priority_queue
   typedef Compare_     Compare;
   typedef ID_          ID ;
   
-  #ifdef CGAL_SURFACE_MESH_SIMPLIFICATION_USE_RELAXED_HEAP
+#ifdef CGAL_SURFACE_MESH_SIMPLIFICATION_USE_RELAXED_HEAP
```

Comment:
is the relaxed heap now officially documented in boost? I wonder whether we should add as template parameter the possibility to have the relaxed heap. If the performance gain is noticeable it might be interesting for some application (those targeting an approximation error for example)

Reply:
No, it is still in /boost/pending/pending